### ISSUE TITLE
release tarball is missing one file in test/integration/targets/unarchive/files/

### ISSUE THREAD

**Issue by pcahyna** [ansible/ansible#28519]
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

release process
##### ANSIBLE VERSION

```
ansible 2.3.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.5 (default, May  3 2017, 07:55:04) [GCC 4.8.5 20150623 (Red Hat 4.8.5-14)]
```

##### OS / ENVIRONMENT

RHEL 7.4
##### SUMMARY

Ansible sources contain the file `test/integration/targets/unarchive/files/test-unarchive-nonascii-くらとみ.tar.gz`. It is not present in the source tarball from http://releases.ansible.com/ansible/ , causing the integration test for unarchive to fail.
##### STEPS TO REPRODUCE

```
test/runner/ansible-test integration posix/ci/ --start-at unarchive
```
alternatively,
```
curl http://releases.ansible.com/ansible/ansible-2.3.2.0.tar.gz | tar tzvf - | grep integration/targets/unarchive/files
```
Interestingly, the source tarball from GitHub ( https://github.com/ansible/ansible/releases ) has the file. Compare the output of
```
curl https://codeload.github.com/ansible/ansible/tar.gz/v2.3.2.0-1  | tar tzvf - | grep integration/targets/unarchive/files
```
Therefore, the problem most likely does not lie in GitHub but in the process of creating the release tarballs for the Ansible website.

##### EXPECTED RESULTS

The integration test should succeed; the output of the pipeline shall show three lies, the last one the filename with hiragana characters
```
drwxrwxr-x root/root         0 2017-08-04 22:24 ansible-2.3.2.0-1/test/integration/targets/unarchive/files/
-rw-rw-r-- root/root         8 2017-08-04 22:24 ansible-2.3.2.0-1/test/integration/targets/unarchive/files/foo.txt
-rw-rw-r-- root/root      4947 2017-08-04 22:24 ansible-2.3.2.0-1/test/integration/targets/unarchive/files/test-unarchive-nonascii-くらとみ.tar.gz
```

##### ACTUAL RESULTS

Integration test fails:

```
TASK [unarchive : test that unarchive works with an archive that contains non-ascii filenames] ***
fatal: [testhost]: FAILED! => {"changed": false, "failed": true, "msg": "Unable to find 'test-unarchive-nonascii-くらとみ.tar.gz' in expected paths."}
        to retry, use: --limit @/root/rpmbuild/BUILD/ansible-2.3.2.0/test/integration/unarchive-WXuhM0.retry

PLAY RECAP *********************************************************************
testhost                   : ok=59   changed=33   unreachable=0    failed=1   

NOTICE: To resume at this test target, use the option: --start-at unarchive
NOTICE: To resume after this test target, use the option: --start-at unicode
ERROR: Command "ansible-playbook unarchive-WXuhM0.yml -i inventory -e @integration_config.yml" returned exit status 2.
```

The pipeline shows that the file is missing:
```
drwxr-xr-x root/root         0 2017-08-07 18:31 ansible-2.3.2.0/test/integration/targets/unarchive/files/
-rw-r--r-- root/root         8 2017-08-07 18:22 ansible-2.3.2.0/test/integration/targets/unarchive/files/foo.txt
```

**Comment by bcoca** [comment]
cc @jimi-c, @mattclay

**Comment by mattclay** [comment]
cc @abadger

**Comment by abadger** [comment]
This has been fixed in devel as a side effect of eff757eeb8e54022a3ae91752b6a09e3d30c5e04

distutils is able to include that file fine but setuptools is not.  The new setup.py code uses distutils so this works now.  Closing as the 2.4.x tarballs will now include this file.
